### PR TITLE
Add an intro to plugins and the plugin manager

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
This is sort of related to #36 
I thought it would be good to have a dedicated page to plugins and installing them if someone is running a real intro workshop.

### CODE REVIEW

**Comment by psobolewskiPhD** [comment]
Thanks @melissawm !
I'll try to get on this asap, but I will be taking some PTO, then I'm at I2K, and then PTO again, so I might slip till the first week of Nov. If anyone wants to push to this branch or merge and iterate feel free!

**Comment by melissawm** [comment]
I can put these suggestions in, and we can follow up with any other notes later. Thanks!

File: napari-workshops/plugins.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+it will attempt to install plugins via `pip`. You can control this using the `Installation Info` dropdown 
+in the Plugin Manager.  
+
+Also keep in mind that plugins with complex or compiled dependencies or those that require non-Python components may not install properly. In those cases, you will need to check the documentation of the plugin for information regarding installation.
+```
+
+For more information on plugins, please see the [official Plugin documentation](https://napari.org/stable/plugins/index.html).
+
+## Install Your First Plugins  
+* Navigate to [napari-hub.org](https://www.napari-hub.org)—this is the platform to discover and share plugins.
```
Comment (melissawm) [inline napari-workshops/plugins.md:27]:
```suggestion
* (Optional) Navigate to [napari-hub.org](https://www.napari-hub.org)—this is the platform to discover and share plugins. Actual plugin installation will be done via napari--we detail this step below.
```

File: napari-workshops/plugins.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    for the `labels` layer.
+    * Use the checkboxes to select some properties to measure, such as `perimeter` or `position`. Refer to the [scikit-image `measure.regionprops` documentation](https://scikit-image.org/docs/stable/api/skimage.measure.html#skimage.measure.regionprops) for more information
+    * Click run, which should yield a table of measurements—depending how many you checked, this could take
+    a moment to run.
+
+    ```{tip}
+    Grab the titlebar of the new widget `Properties of nuclei label` and drag it off the napari
+    viewer to undock. Then you can resize the table for easier browsing.
+    You can re-dock the widget by dragging it on the existing side dock  or the bottom of the viewer,
+    e.g. where the viewer buttons are, to dock it beneath the canvas.
```
Comment (melissawm) [inline napari-workshops/plugins.md:67]:
```suggestion
    e.g. where the viewer buttons are, to dock it as a tab next to "layer list".
```

**Review by melissawm — APPROVED** [review]

File: napari-workshops/plugins.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    for the `labels` layer.
+    * Use the checkboxes to select some properties to measure, such as `perimeter` or `position`. Refer to the [scikit-image `measure.regionprops` documentation](https://scikit-image.org/docs/stable/api/skimage.measure.html#skimage.measure.regionprops) for more information
+    * Click run, which should yield a table of measurements—depending how many you checked, this could take
+    a moment to run.
+
+    ```{tip}
+    Grab the titlebar of the new widget `Properties of nuclei label` and drag it off the napari
+    viewer to undock. Then you can resize the table for easier browsing.
+    You can re-dock the widget by dragging it on the existing side dock  or the bottom of the viewer,
+    e.g. where the viewer buttons are, to dock it as a tab next to "layer list".
```
Comment (psobolewskiPhD) [inline napari-workshops/plugins.md:67]:
```suggestion
    by dragging the widget title bar from below the viewer onto the canvas (left of the `Activity` button).
```

File: napari-workshops/plugins.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    for the `labels` layer.
+    * Use the checkboxes to select some properties to measure. Refer to the [scikit-image `measure.regionprops` documentation](https://scikit-image.org/docs/stable/api/skimage.measure.html#skimage.measure.regionprops) for more information
+    * Click run, which should yield a table of measurements—depending how many you checked, this could take
+    a moment to run.
+
+    ```{tip}
+    Grab the titlebar of the new widget `Properties of nuclei label` and drag it off the napari
+    viewer to undock. Then you can resize the table for easier browsing.
+    You can re-dock the widget by dragging it on the existing side dock  or the bottom of the viewer,
+    e.g. where the viewer buttons are, to dock it beneath the canvas.
```
[the remark this answers is not in the document]
Comment (psobolewskiPhD) [reply napari-workshops/plugins.md]: Yeah this isn't working, it's tricky. I made a new commit to better describe it. works for me to drag the widget title bar from below the viewer onto the canvas.

**Comment by melissawm** [comment]
Thanks @psobolewskiPhD !

**Pull request merged**